integration_tests/fixtures/activitiesApi/getScheduledInstance93.json

```diff
@@ -60,7 +60,7 @@
       "posted": false,
       "recordedTime": null,
       "recordedBy": null,
-      "status": "WAIT",
+      "status": "WAITING",
       "payAmount": null,
```

Comment:
I have a feeling there's some confusion over the attendance status in recent PRs? This was the latest https://github.com/ministryofjustice/hmpps-activities-management-api/pull/222 (UI) which has an enum for attendance status: ```enum class AttendanceStatus {
  CANCELLED,
  COMPLETED,
  WAIT,
}```

Reply:
I had though the statuses were now  `WAITING`, `COMPLETED`, `LOCKED`   (Cancelled is an outcome reason rather than a status). Confused now, and probably worth a talk through tomorrow?

Reply:
Discussed and agreed